jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/action/ExportAction.java

```diff
@@ -95,6 +104,65 @@ public void setGridExportProperties(GridExportProperties gridExportProperties) {
         this.availableExportModes = gridExportProperties.getDefaultExportModes().stream()
                 .map(ExportMode::valueOf)
                 .toList();
+        this.columnExportFilter = ColumnExportFilter.valueOf(gridExportProperties.getDefaultColumnExportFilter());
+    }
+
+    /**
+     * Sets the {@link ColumnExportFilter} that is used to filter columns to export. This is a simple, predefined
+     * alternative to {@link #setColumnExportPredicate(Predicate)}. Has the lowest filtering priority.
+     * <p>
+     * The default value depends on {@link GridExportProperties#getDefaultColumnExportFilter()}.
+     *
+     * @param columnExportFilter column export filter to set
+     */
+    public void setColumnExportFilter(ColumnExportFilter columnExportFilter) {
+        this.columnExportFilter = columnExportFilter;
+    }
+
+    /**
+     * @return this
+     * @see #setColumnExportFilter(ColumnExportFilter)
+     */
+    public ExportAction withColumnExportFilter(ColumnExportFilter columnExportFilter) {
+        setColumnExportFilter(columnExportFilter);
+        return this;
+    }
+
+    /**
+     * Sets the {@link Predicate} that is used to filter columns to export. This is a flexible alternative to
+     * {@link #setColumnExportFilter(ColumnExportFilter)}. Has secondary filtering priority.
+     *
+     * @param columnExportPredicate column export predicate to set
+     */
+    public void setColumnExportPredicate(Predicate<Grid.Column<Object>> columnExportPredicate) {
```

Comment:
Let's name it `setColumnExportFilter`. Currently, without reading JavaDoc, it's unclear what we exactly set.